cac-sandbox.yml

```diff
@@ -2,6 +2,24 @@ jenkins:
   systemMessage: >-
     This jenkins is managed by configuration as code, update <a
     href="https://github.com/hmcts/cnp-jenkins-config/blob/master/cac-sandbox.yml">here</a>
+  globalNodeProperties:
+    - envVars:
+        env:
+          - key: INFRA_VAULT_NAME
+            value: infra-vault-sandbox
+          - key: NONPROD_ENVIRONMENT_NAME
+            value: saat
+          - key: NONPROD_SUBSCRIPTION_NAME
+            value: sandbox
+          - key: PROD_ENVIRONMENT_NAME
+            value: saat
```

Comment:
maybe a comment on why?

Reply:
was debating putting a comment but think they're both going to change to `sandbox` soon anyway